app/steps/ui/deceased/dob/index.js

```diff
@@ -14,7 +15,22 @@ class DeceasedDob extends DateStep {
     }
 
     handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
+        const dod = new Date(`${session.form.deceased.dod_year}-${session.form.deceased.dod_month}-${session.form.deceased.dod_day}'`);
+        dod.setHours(0, 0, 0, 0);
+
+        const dob = new Date(`${ctx.dob_year}-${ctx.dob_month}-${ctx.dob_day}'`);
```

Comment:
It looks like there is an additional `'` at the end of the date string

Reply:
That's removed